### PR TITLE
Some modifications on coala.io/tutorial

### PR DESCRIPTION
**Pull request by Asnelchristian**

### CODE REVIEW

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ Prerequisites
 -------------
 
 In order to complete this tutorial you will need coala installed.
-(Installation is actually not required but it's more convenient and
-recommended.)
+You can find the `instructions for the installion here
+<http://docs.coala.io/en/latest/Users/Install.html>`_.
```
Comment (sils) [inline Users/Tutorial.rst]: https://coala.io/install always use shortlinks when possible

File: Users/Tutorial.rst [hunk 2]
```diff
@@ -83,6 +83,15 @@ the end of the line, after ``#include <stdio.h>`` in the ``main.c`` file. coala
...
 patch (option 2).
 
+You can also run coala in non interactive mode (given that all the settings
+required by the bears you are using are provided in the ``.coafile``)
+
+::
+
+    coala  --non-interactive
+
+In this case there won't be any interaction, the patch will be apply directly.
```
Comment (sils) [inline Users/Tutorial.rst]: it won't be applied, it'll be shown. This is meant for continuous integration e.g.

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/409/commits/9f30e6e3963cf8fb941329e32f0d48fe1eed6cd1 https://github.com/coala/documentation/pull/409/commits/4cd425541de200fa1fd353880f58abb6e9d0c658

**Comment by sils** [comment]
unack https://github.com/coala/documentation/pull/409/commits/6f1c9f4a4a86273c99802d685a3a5f619ad92404

File: Users/Tutorial.rst (hunk 1 above)
Comment (sils) [inline Users/Tutorial.rst]: can we shorten those two sentences to one

**Comment by Asnelchristian** [comment]
:+1:

2017-02-01 19:42 GMT+01:00 Lasse Schuirmann <notifications@github.com>:

**Comment by Asnelchristian** [comment]
reack https://github.com/coala/documentation/pull/409/commits/b7cdd62d965c4e461a844dadd9751d034923368c https://github.com/coala/documentation/pull/409/commits/0f9b4feacc7f47bc5ac1564009b73c9efe0c0d36

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/409/commits/ac98bb8fc964aece800a324582de75bed242fda3

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/409/commits/b3dd813453f9cc051bc681d8f11ce3a03c58ff22

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/409/commits/8c397f54193da1208a53672b856b80346943e25a

**Comment by sils** [comment]
@rultor merge

**Pull request merged**